css/style.css

```diff
@@ -224,6 +224,10 @@ footer p {
   display: none;
 }
 
+.hide1 {
```

Comment:
I assume you were just experimenting here. You could have just used the initial `hide` class without needing to add an additional one.